learn/advanced/asynchronous_operations.md

```diff
@@ -118,17 +140,95 @@ Had the task failed, the response would have included an `error` object:
 }
 ```
 
+If the task had been canceled while it was `enqueued` or `processing`, it would have the `canceled` status and a non-`null` value for the `canceledBy` field:
+
+```json
+{
+  "uid": 1,
+  "indexUid": null,
+  "status": "canceled",
+  "type": "taskCancelation",
```

Comment:
Here we represent a taskCancelation that another taskCancelation has canceled. That's a bit meta 😎. To be closer to what will happen most of the time, I would put a documentAdditionOrUpdate or a settingsUpdate.